plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -28,11 +28,14 @@
 public class StaticMetastoreConfig
 {
     public static final String HIVE_METASTORE_USERNAME = "hive.metastore.username";
+    public static final String HIVE_METASTORE_CATALOG_NAME = "hive.metastore.catalog-name";
 
     private static final Splitter SPLITTER = Splitter.on(',').trimResults().omitEmptyStrings();
 
     private List<URI> metastoreUris;
     private String metastoreUsername;
+    // HiveMetastore uses "hive" as the catalog, therefore setting here as the default one to make it backward compatible
```

Comment:
no need to set a default value when the `catalogName` is nullable - in case you accept my suggestion not to modify the functionality of the existing `ThriftHiveMetastoreClient`

Reply:
>in case you accept my suggestion not to modify the functionality of the existing ThriftHiveMetastoreClient

sorry, did not understand it exactly, are we saying we do not need to append the the `catalogName` to the `databaseName` at all in the `ThriftHiveMetastoreClient` or need to set only if the catalogName is not "hive" ?